refactor: Replaced telerik components with oxyplot in Views, …

ramateur opened the pull request:
…except AnalyticReports

drusteeby reviewed (commented):
LGTM. Just gotta remove app.config changes

ramateur commented on App.config:
Tell me please how to do it. Add it to gitignore?

drusteeby replied:
Copy and paste the original and push it as a new commit.

ramateur replied:
Done

drusteeby commented:
This should remain open until @silahian has a chance to merge it.

silahian commented:
before I merge this, @ramateur can you please make sure to follow these instructions:
https://github.com/silahian/VisualHFT/issues/2#issuecomment-1646988971
And then chack that all is ok?

ramateur commented:
Ok, Ithink it can take a couple of days. I added dummy data to database, but launch cound not analytics report with it, it still displays No data (I changed date picker as you described) (
Maybe it is because I had to switch to Binance povider instead of BinanceUS. The former is not accessible for me. And switched pair to BTC-USDT, which is present both on Coinbase and Binance

ramateur commented:
There is a bug in demoTradingCore Program.cs when chagnging BinanceUs provider to Binance, symbol pair to "BTC-USDT" and commenting string 31 //await InitializeCoinbase(). Occurs mismatch between key-value pair in dictionary. I worked it around replacing book.MarketSymbol with _SYMBOLS.First() value as key, but may be you have another point of view on it.
